src/open_inwoner/accounts/views/cases.py

```diff
@@ -145,27 +131,67 @@ def get_cases(self):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        cases = self.get_cases()
+        raw_cases = self.get_cases()
+        cases = self.process_cases(raw_cases)
+        context["cases"] = cases
 
-        context["anchors"] = [
-            ("#pending_apps", _("Lopende aanvragen")),
-            ("#completed_apps", _("Afgeronde aanvragen")),
-        ]
+        context["anchors"] = self.get_anchors()
+        context["title"] = self.get_title()
+        return context
+
+    def get_anchors(self) -> list:
+        return []
+
+    def get_title(self) -> str:
+        return ""
+
+
+class OpenCaseListView(
+    BaseBreadcrumbMixin, CaseAccessMixin, CaseListMixin, TemplateView
+):
+    @cached_property
+    def crumbs(self):
+        return [(_("Mijn aanvragen"), reverse("accounts:my_open_cases"))]
+
+    def get_cases(self):
+        all_cases = super().get_cases()
+
+        cases = [case for case in all_cases if not case.einddatum]
+        cases.sort(key=lambda case: case.startdatum, reverse=True)
+        return cases
 
-        context["open_cases"] = [
-            case
-            for case in cases
-            if not case["end_date"] and not case["current_status"] == "Afgerond"
+    def get_anchors(self) -> list:
+        return [
+            ("#cases", _("Lopende aanvragen")),
+            (reverse("accounts:my_closed_cases"), _("Afgeronde aanvragen")),
         ]
-        context["open_cases"].sort(key=lambda case: case["start_date"])
-        context["closed_cases"] = [
-            case
-            for case in cases
-            if case["end_date"] or case["current_status"] == "Afgerond"
+
+    def get_title(self) -> str:
+        return _("Lopende aanvragen")
+
+
+class ClosedCaseListView(
+    BaseBreadcrumbMixin, CaseAccessMixin, CaseListMixin, TemplateView
+):
+    @cached_property
+    def crumbs(self):
+        return [(_("Mijn aanvragen"), reverse("accounts:my_closed_cases"))]
+
+    def get_cases(self):
+        all_cases = super().get_cases()
+
+        cases = [case for case in all_cases if case.einddatum]
+        cases.sort(key=lambda case: case.einddatum, reverse=True)
+        return cases
+
+    def get_anchors(self) -> list:
+        return [
+            (reverse("accounts:my_closed_cases"), _("Lopende aanvragen")),
```

Comment:
This should redirect to `"accounts:my_open_cases"` , right?Now both anchors redirect to closed cases.

Reply:
Nice catch!